### DISCUSSION TITLE
Documentation on berry

### DISCUSSION THREAD

**Discussion by waltkind** [arendst/Tasmota#20274] [Q&A] [answered]
Is there a more actual documentation of the berry scripting language?
What I found on github was `The Berry Script Language
Reference Manual
V0.1.3
Guan Wenliang
August 5, 2020` 
and the wiki `berry
Release 4.9.0
Guan Wenliang & Stephan Hadinger
Jul 29, 2023`.

On searching the internet I found a new command `string.replace()` that s-hadinger merged on jun 24, 2022 that is not mentioned in this documents.

Perhaps there are some more things new, that I haven't found yet.

I know documentation is hard and this is no rant, it is a question.

**Comment by s-hadinger** [comment]
The (almost) up to date documentation is in the wiki 
https://github.com/berry-lang/berry/wiki/Reference

Indeed some updates were forgotten.

**Reply by sfromis** [reply, marked as answer]
I've occasionally added a few updates, wiki-style open to anyone to improve on.
